### PR TITLE
💎(site) Add New Design to Technical Page Hero

### LINKED ISSUE #338: Module: Hero (Technical)
Technical Landing Page :: Hero

Design: https://www.figma.com/file/xtBKZcRBo7H2Jrjs7DjNqE/Final-Designs?node-id=2%3A34277

### PR DESCRIPTION
**Pull request by Vanss472**
Added new design to technical page hero.

Hero notes: 
1. Since the image should appear on the right side of the screen, I have removed the position variable from markup.
2. I used the section modifier to target each collection and apply the respective hero color
3. I'll need the final path for the SVGs images `ix://landings/technical/`

- Resolves #338

### CODE REVIEW

File: site/sass/layouts/_tech-detail.scss [hunk 1]
```diff
@@ -36,36 +39,54 @@
...
+
+      @include mq($bkpt-tech-details--2-cols) {
+        height: 100%;
+        padding-left: 0;
+      }
 
-      @include mq($bkpt-tech-details--landing) {
-        padding-bottom: 4rem;
+      @include mq($bkpt-tech-details--3-cols) {
+        height: 40vh;
```
Comment (Snugug) [inline site/sass/layouts/_tech-detail.scss:84]: I think this needs to be `max-height` or the background image starts to break

```suggestion
        max-height: 40vh;
```

File: site/en/linux/index.md [hunk 2]
```diff
@@ -2,8 +2,7 @@
 title: Overview
```
Comment (Snugug) [inline site/en/linux/index.md:2]:
```suggestion
title: Linux on Chrome OS
```

Like the change you made to Productivity. Let's do the same for Linux

File: site/sass/layouts/_tech-detail.scss [hunk 3]
```diff
@@ -36,36 +39,54 @@
...
+    &-title {
+      color: var(--white);
+      padding: $block-padding clamp(1.5rem, 4vw, 7.5rem);
+
+      @include mq($bkpt-tech-details--2-cols) {
+        align-items: center;
+        display: flex;
+        justify-content: center;
+        padding-bottom: 0;
+        padding-right: 0;
```
Comment (Snugug) [inline site/sass/layouts/_tech-detail.scss:67]: I think it needs to retain this padding or lest the image and text get too close

File: site/_layouts/tech-detail.njk [hunk 4]
```diff
@@ -2,43 +2,38 @@
...
     {% endif %}
   </section>
-
-  {% if landingPage == false %}
-  <section class="tech-detail__recommended wrapper--padding wrapper--padded wrapper__contained">{{itemGrid(microcopy.recommended, title | related(locale.code, page.url))}}</section>
-  {% endif %}
 </article>
-{% from 'back-to-top.njk' import backToTop %} {{ backToTop() }}
+{% if landingPage == false %}
+<section class="tech-detail__recommended wrapper wrapper--padding wrapper--padded wrapper--full-bleed wrapper__contained wrapper__background">{{itemGrid(microcopy.recommended, title | related(locale.code, page.url))}}</section>
```
Comment (Snugug) [inline site/_layouts/tech-detail.njk:38]:
```suggestion
<aside class="tech-detail__recommended wrapper wrapper--padding wrapper--padded wrapper--full-bleed wrapper__contained wrapper__background">{{itemGrid(microcopy.recommended, title | related(locale.code, page.url))}}</aside>
```
This probably should have been an `aside` to begin with, but def should be one now

**Review by Snugug — CHANGES_REQUESTED** [review]
Thanks! This is all looking great! I've got a couple of comments in-line, but the incoming need to swap images for single and double columns may mean the hero image stuff needs to change to the theme stuff we talked about. TBD.

In addition to the inline comments, I've got one more:
* Below about 470px the background image starts to become detached from the bottom right corner

Thanks!

File: site/sass/layouts/_tech-detail.scss (hunk 3 above)
Comment (Vanss472) [reply site/sass/layouts/_tech-detail.scss]: Fixed.

**Review by Snugug — DISMISSED** [review]

**Review by Snugug — APPROVED** [review]

**Comment by Snugug** [comment]
CI is stalled. Going to merge.

**Pull request merged**